prep-html-links/contact-us/home-page.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Home Page</title>
+</head>
+<body>
+  <h1>Home Page</h1>
+  <a href= "file:///Users/jeffreyneiljuarez/repos/prep-0421-code-solutions/prep-html-links/contact-us/contact-us.html">Click here to go to Contact Us</a>
```

Comment:
You should use a relative path here instead of an absolute path to the file on your computer.